doc/modules/clustering.rst

```diff
@@ -1339,6 +1339,91 @@ mean of homogeneity and completeness**:
    <http://www.cs.columbia.edu/~hila/hila-thesis-distributed.pdf>`_, Hila
    Becker, PhD Thesis.
 
+.. _fowlkes_mallows_scores:
+
+Fowlkes-Mallows scores
+----------------------
+
+The Fowlkes-Mallows index (FMI) is defined as the geometric mean between of
+the precision and recall::
+
+      FMI = TP / sqrt((TP + FP) * (TP + FN))
+
+Where :math:`TP` is the number of **True Positive** (i.e. the number of pair
+of points that belong to the same clusters in both labels_true and
+labels_pred), :math:`FP` is the number of **False Positive** (i.e. the number
+of pair of points that belong to the same clusters in labels_true and not
+in labels_pred) and :math:`FN`is the number of **False Negative** (i.e the
+number of pair of points that belongs in the same clusters in labels_pred
+and not in labels_True).
+
+The score ranges from 0 to 1. A high value indicates a good similarity
+between two clusters.
+
+  >>> from sklearn import metrics
+  >>> labels_true = [0, 0, 0, 1, 1, 1]
+  >>> labels_pred = [0, 0, 1, 1, 2, 2]
+
+  >>> metrics.fowlkes_mallows_score(labels_true, labels_pred)  # doctest: +ELLIPSIS
+  0.47140...
+
+One can permute 0 and 1 in the predicted labels, rename 2 to 3 and get
+the same score::
+
+  >>> labels_pred = [1, 1, 0, 0, 3, 3]
+
+  >>> metrics.fowlkes_mallows_score(labels_true, labels_pred)  # doctest: +ELLIPSIS
+  0.47140...
+
+Perfect labeling is scored 1.0::
```

Comment:
Yes, I recall that this already happens in the file. I may propose an issue to make it more succinct.